index.js

```diff
@@ -104,7 +105,7 @@ function checkOptionUsage(config, token) {
     throw new ERR_PARSE_ARGS_INVALID_OPTION_VALUE(`Option '${shortAndLong} <value>' argument missing`);
   }
   // (Idiomatic test for undefined||null, expecting undefined.)
-  if (type === 'boolean' && token.value != null) {
+  if (type === 'boolean' && token.value != null && !token.isDefaultValue) {
```

Comment:
`isDefaultValue` is no longer being used

Reply:
Done